lib/go-atscfg/regexremapdotconfig.go

```diff
@@ -108,19 +108,19 @@ func deliveryServicesToCDNDSes(dses []DeliveryService) (map[tc.DeliveryServiceNa
 	warnings := []string{}
 	sDSes := map[tc.DeliveryServiceName]cdnDS{}
 	for _, ds := range dses {
-		if ds.OrgServerFQDN == nil || ds.QStringIgnore == nil || ds.XMLID == nil {
-			if ds.XMLID == nil {
+		if ds.OrgServerFQDN == nil || ds.QStringIgnore == nil || &ds.XMLID == nil {
+			if &ds.XMLID == nil {
```

Comment:
Instead of checking `&ds.XMLID == nil`, check `ds.XMLID == ""`.

Reply:
done

Reply:
done